plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixConfig.java

```diff
@@ -16,16 +16,22 @@
 import com.google.common.base.Splitter;
 import com.google.common.collect.ImmutableList;
 import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
 import io.airlift.configuration.validation.FileExists;
 
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
 import javax.validation.constraints.NotNull;
 
 import java.util.List;
 
 public class PhoenixConfig
 {
+    static final int MAX_ALLOWED_SCANS_PER_SPLIT = 1000;
```

Comment:
Add some code comment (and commit description) with regards to significance of this value.

Reply:
In theory no upper limit is needed. When this is set to a very large value Trino is going to have one split per HBase region.
I'll add a comment.